api/src/graphql/schema.ts

```diff
@@ -4,10 +4,12 @@ import { validationPlugin } from 'nexus-validation-plugin';
 import { applyMiddleware } from 'graphql-middleware';
 import * as types from '../modules';
 import { GQLDateBase } from './dateSchema';
+import * as demoTypes from 'powerhouse-switchboard-plugin-demo';
+import * as anotherDemoTypes from 'powerhouse-switchboard-plugin-demo-2';
```

Comment:
This looks like magic and doesn't give me (or for sure any other developer) a sense of what needs to be exported from the package to make it work? Can you please avoid using `*`? 

Ideally, as I proposed in the issue `setup()` (which is currently called in the other file) would return required types (so you import something from the package once and call it once as well)

Reply:
see the change.